code/modules/requisitions/_supply_export.dm

```diff
@@ -0,0 +1,23 @@
+/atom/movable/proc/supply_export()
+	return 0
+
+
+/obj/structure/closet/crate/supply_export()
+	. = 3
+	SSpoints.supply_points += .
+
+
+/mob/living/carbon/xenomorph/supply_export()
+	switch(tier)
+		if(XENO_TIER_ZERO)
+			SSpoints.supply_points++
+			. = 1

```

Comment:
Change this to
```
			. = 2
```
and remove the line above.

Reply:
Forgot to remove that when I changed the system. They were supposed to net a single point, yea.
This is for larvas.